dlt/helpers/airflow_helper.py

```diff
@@ -163,6 +168,9 @@ def add_run(
                               will remain sequential. Use another executor, e.g. CeleryExecutor)!
                         NOTE: The first component of the source is done first, after that
                               the rest are executed in parallel to each other.
+                    parallel-isolated - decompose the source into a parallel Airflow task group.
+                        NOTE: In case the SequentialExecutor is used by Airflow, the tasks
```

Comment:
please explain that we run each task in a pipeline with different (but derived) name to separate the state